src/app/main/main.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component } from '@angular/core';
+import { Ad } from 'types/ad';
+
+@Component({
+  selector: 'app-main',
+  templateUrl: './main.component.html',
+  styleUrls: ['./main.component.css']
+})
+export class MainComponent{
+
+  //data sample
+  ads: Ad[]=[
```

Comment:
move all data into separate service

Reply:
try to follow this patterns https://medium.com/@ankit.sinhal/mvc-mvp-and-mvvm-design-pattern-6e169567bbad